.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -442,6 +459,31 @@ for (const serverVersion of SERVER_VERSIONS) { %>
     commands:
       - func: prepare
       - func: install
+
+<% for (const browser of BROWSERS) { %>
+  <% for(const group of E2E_TEST_GROUPS) { %>
+  - name: test-web-sandbox-<%= browser %>-<%= group.number %>
+    tags: ['required-for-publish', 'run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+        vars:
+          scope: 'compass-e2e-tests'
+      - func: apply-compass-target-expansion
+        vars:
+          compass_distribution: compass
+      - func: test-web-sandbox
+        vars:
+          mongodb_version: latest
+          browser_name: '<%= browser %>'
+          compass_distribution: compass
+          e2e_test_groups: <%= E2E_TEST_GROUPS.length %>
+          e2e_test_group: <%= group.number %>
+          debug: 'compass-e2e-tests*,electron*,hadron*,mongo*'
+  <% } %>
+<% } %>
+
```

Comment:
You're inserting this task in the middle of another task, that's why it's running functions it's not supposed to 🙂

Reply:
fixed, i hope